workshop/Lab5/README.md

```diff
@@ -0,0 +1,227 @@
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-2     redis        Open source, advanced            redis-4-0-2
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-6     redis        Open source, advanced            redis-4-0-6
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-7     redis        Open source, advanced            redis-4-0-7
+                         key-value store. It i
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-8     redis        Open source, advanced            redis-4-0-8
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+  4-0-9     redis        Open source, advanced            redis-4-0-9
+                         key-value store. It is
+                         often referred to as a data
+                         structure server since keys
+                         can contain strings, hashes,
+                         lists, sets and sorted sets.
+```
+
+The numbered tuples are the version of the helm chart minibroker uses to deploy the services we ask it to provison.
+For the purposes of this workshop we'll be using the redis Class and the 4-0-9 plan.
+
+Use svcat to provision an instance by running 
+```console
+$ svcat provisison myredis --class=redis --plan=4-0-9
+  Name:        myredis  
+  Namespace:   default   
+  Status:                
+  Class:       redis     
+  Plan:        4-0-9
+```
+
+If you're interested in a more in-depth explanation of what's going on behind the scenes, check [here](provision.md).
+After provisioning, you should be able to list service instances and see your instance:
+```console
+$ svcat get instances
+   NAME     NAMESPACE   CLASS   PLAN    STATUS
++---------+-----------+-------+-------+--------+
+  myredis   default     redis   4-0-9   Ready
+```
+
+Also, just to make sure it bound to your persistant volume correctly, get all pods and
+make sure the new redis pods are up and running. Minibroker provisions them with a randomly
+generated name, in this case "jump-albatross".
```

Comment:
Its kind of odd they're in the default namespace instead of minbroker's isn't it?

Reply:
i guess? open an issue on minibroker